### DISCUSSION TITLE
[PSD11] Exercise 3.2

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#172] [Esercizi (A-L)]
![image](https://user-images.githubusercontent.com/69389461/210061785-17a4f04d-7208-41c4-a3be-1ce396de72f5.png)

**Comment by Elia-Belli** [comment]
![d](https://user-images.githubusercontent.com/107944476/213930326-519e0156-36bf-4c90-bed1-756ce5ea1736.png)

La $X$ è una Contention, causata da un segnale invalido.

**Comment by fraxb04** [comment]
@Elia-Belli 
1) non capisco se Q inizia a scendere quando (con S= 0) R raggiunge l'apice della salita, oppure quando R inizia la sua salita.
2) non capisco se Q inizia a salire quando (con R = 0) S raggiunge l'apice della salita, oppure quando inizia la sua salita.

- da come posso vedere, interpreto per il punto 1, quando raggiunge l'apice, mentre per il punto 2, quando inizia la salita

**Comment by Elia-Belli** [comment]
In generale $Q$ cambia valore poco dopo che $R$ o $S$ cambiano valore (finiscono la salita)

**Comment by FeddyLix17** [comment]
@Elia-Belli a me la forma d'onda viene praticamente identica alla tua, ma ad esempio se **Q**=**Q<sub>p</sub>**=1, **S**=0, **R**=0; **Q** non dovrebbe iniziare la sua discesa pochissimo dopo che R ha iniziato la sua salita?

**Comment by Elia-Belli** [comment]
> @Elia-Belli a me la forma d'onda viene praticamente identica alla tua, ma ad esempio se **Q**=**Qp**=1, **S**=0, **R**=0; **Q** non dovrebbe iniziare la sua discesa pochissimo dopo che R ha iniziato la sua salita?

No, perchè il Reset avviene quando $R=1$ e poco dopo l'inizio della salita $R$ è ancora un valore intermedio, finita la salita $R=1$ e con un piccolo delay $Q$, accorgendosi del segnale, inizia la sua discesa.
